native-libs/deps/recipes/mymonerocorecpp/mymonerocorecpp.recipe

```diff
@@ -15,6 +15,7 @@ build() {
 #    patch -p1 < $recipe_dir/ios-build.patch
 #    patch -p1 < $recipe_dir/timeout.patch
 
+    rm -rf contrib/monero-core-custom
```

Comment:
When using local `monero-core-custom` there might be issues with copying duplicate git objects.